tensorrt_llm/_torch/pyexecutor/resource_manager.py

```
@@ -3073,6 +3074,19 @@ def resume_request(self, req: LlmRequest) -> bool:
             return False
         return self._resume_and_restore(req.py_request_id, kv_cache)
 
+    def prefetch_for_context_tokens(self, requests: list) -> None:
+        """Prefetch radix-tree blocks from disk→host for upcoming context requests."""
+        if not self.enable_block_reuse:
+            return
+        HOST_LEVEL = CacheLevel(1)
```

Ruff (0.15.15)

[error] Undefined name `CacheLevel`

(F821)